plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/graphical/edit/policies/CreationUtil.java

```diff
@@ -91,40 +85,56 @@ public class CreationUtil {
      * @param realLocation
      *            the location of the clicked point.
      * @param editPart
-     *            the edit part
+     *            the edit part on which the calling policy is installed.
      * @since 0.9.0
      */
     public CreationUtil(final CreateRequest request, final IDiagramCommandFactory commandFactory, final Point realLocation, final EditPart editPart) {
-        this.realLocation = realLocation;
-        this.realSize = null;
-        this.request = request;
-        this.emfCommandFactory = commandFactory;
-        this.editPart = editPart;
+        // The size of the request take into account the zoom (got the size in 100%)
+        this(commandFactory, new RootLayoutData(editPart, realLocation.getCopy(), CreationUtil.adaptRequestSizeToZoomFactor(request, editPart)), editPart);
     }
 
     /**
      * Creates a new <code>CreationUtil</code> with the specified request and location.
```

Comment:
there is no request anymore; update javadoc

Reply:
Not sure to understand your comment. The request is used in method `CreationUtil.adaptRequestSizeToZoomFactor(request...`

Reply:
You're right, I must have missed something the first time.